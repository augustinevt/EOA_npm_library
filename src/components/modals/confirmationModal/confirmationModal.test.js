import React from 'react';
import renderer from 'react-test-renderer';

import ConfirmationModal from './ConfirmationModal';

test('Confirmation default snapshot', () => {
  const snapshot = renderer.create(
    <ConfirmationModal open={true}/>
  ).toJSON();

  expect(snapshot).toMatchSnapshot();
});
