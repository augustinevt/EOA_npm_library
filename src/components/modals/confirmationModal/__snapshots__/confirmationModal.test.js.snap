// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation default snapshot 1`] = `
<div
  className="confirmation-modal"
>
  <div
    className="confirmation-modal__message"
  >
    This is the message
  </div>
  <div
    className="confirmation-modal__buttons"
  >
    <div
      className="confirmation-modal__button"
      onClick={[Function]}
    >
      Yes
    </div>
    <div
      className="confirmation-modal__button"
      onClick={[Function]}
    >
      No
    </div>
  </div>
</div>
`;
